flybywiresim/aircraft
# fix(a380x): reset perf data on fmc reset

Fixes #9019

## Summary of Changes


## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

- Enter some data on the perf pages like V-speeds
- Swap the database to reset the FMC
- Verify that the entered data resets back to default values


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo**, **flybywire-aircraft-a380-842 (4K)** or **flybywire-aircraft-a380-842 (8K)** download link at the bottom of the page
